pymedphys/labs/winstonlutz/interppoints.py

```diff
@@ -128,3 +130,45 @@ def apply_transform(xx, yy, transform):
     yy_transformed.shape = yy.shape
 
     return xx_transformed, yy_transformed
+
+
+def create_bb_points_function(bb_diameter):
+    min_dist = 0.5
+    distances = np.arange(0, bb_diameter * 0.8, min_dist)
+
+    x = []
+    y = []
+    dist = []
+
+    for _, distance in enumerate(distances):
+        (
+            new_x,
+            new_y,
+        ) = pymedphys._utilities.createshells.calculate_coordinates_shell_2d(  # pylint: disable = protected-access
```

Comment:
Just curious, do we have a general rule of thumb as to when we allow access to protected members and when is best to avoid it?

Reply:
In this case `_utilities` is a module private to the `pymedphys` library. Any function within `pymedphys` can use these private modules, any external function should not as they will not be governed by the semver upgrade rules (https://semver.org/).